app/clients/summary_client.py

```
@@ -37,10 +58,50 @@ def __init__(
+            httpx.HTTPError: 기타 HTTP 에러 발생 시.
+        """
+        async with httpx.AsyncClient(timeout=self.timeout) as client:
+            response = await client.post(
+                f"{self.base_url}/summarize/batch",
+                json={"texts": texts},
+            )
+            
+            # 재시도 가능한 상태 코드면 예외 발생
+            if response.status_code in RETRYABLE_STATUS_CODES:
+                raise SummaryServerException(
+                    f"Retryable HTTP error",
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)